cdragontoolbox/export.py

```diff
@@ -434,7 +436,8 @@ def _create_exporter(self, patch):
             TexConverter(),
             BinConverter(re.compile(r'game/.*\.bin$'), btype_version),
             SknConverter(),
-            RstConverter(re.compile(r'game/data/menu/.*\.(txt|stringtable)$'))
+            RstConverter(re.compile(r'game/data/menu/.*\.(txt|stringtable)$')),
+            AtlasInfoConverter(re.compile(r'game/clientstates/.*\.cdtb$'))
```

Comment:
As in, this regex doesn't catch any existing paths or we don't **currently** export those?

The former would be concerning; I'd assume this would be due to the `.cdtb` suffix not getting appended early enough, although I thought it would be. The latter would be correct.

Reply:
Ok; I thought we needed but an converted and a "raw" version but that's not the case. :)